analyzers/its/AllSonarAnalyzerRules.ruleset

```diff
@@ -1152,7 +1152,6 @@
     <Rule Id="S1142" Action="Warning" />
     <Rule Id="S1143" Action="Warning" />
     <Rule Id="S1144" Action="Warning" />
-    <Rule Id="S1145" Action="Warning" />
```

Comment:
this file just has all possible rule IDs from S1 to S10000 - so most of the IDs here do not correspond to actual rules

I think you should revert it to avoid confusion